spec.bs

```diff
@@ -1042,6 +1055,21 @@ dictionary AppHistoryNavigateEventInit : EventInit {
   any info;
 };
 
+dictionary AppHistoryTransitionWhileOptions {
+  AppHistoryFocusReset focusReset;
+  AppHistoryScrollRestoration scrollRestoration;
+};
+
+enum AppHistoryFocusReset {
+  "after-transition",
```

Comment:
No, that would immediately reset the focus (since the focus changed). Then, after the transition completes, if focus hasn't been moved manually, focus would get reset again to the first autofocus="" element or to the body.